lib/node/client.ml

```diff
@@ -21,7 +21,7 @@ let add_peer_as_is node (peer : Peer.t) =
 
 let peers node = Base.Hashtbl.keys node.peers
 
-let create_request node ?(request_ack = false) recipient payload =
+let create_request node ?(request_ack = false) recipient (payload, payload_signature) =
```

Comment:
Since `payload_signature` is represented as an option, why not make it an optional argument?
```ocaml
let create_request node ?(request_ack = false)?payload_signature recipient payload =
```

Reply:
Nice remark, done! 😄